components/Title.jsx

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import Image from "next/image";
+
+import LeftArrow from "../public/LeftArrow-NoCircle.svg";
+import { baseTheme } from "../theme";
+import { Text } from "../components";
+import { media } from "../utils";
+
+/**
```

Comment:
Much better documentation I guess I will start converting mine to this from now on. :)

Reply:
Haha, added it because there's a lot of props